deploy-agent/deployd/client/client.py

```diff
@@ -216,7 +216,7 @@ def send_reports(self, env_reports=None):
                                         availabilityZone=self._availability_zone,
                                         stageType=self._stage_type,
                                         accountId=self._account_id)
-
+                log.info('ping request: %s' % ping_request)
```

Comment:
I think we already log ping request, any reason we log here as well?

Reply:
oh, it is because when I tested locally, I got json error when pinging the agent server since I have not updated on agent server side yet. In this case, I have to log here to see what is the ping request because the code cannot execute to the place that has the log for `ping request -> ping response`